pkg/trimaran/lowriskovercommitment/lowriskovercommitment_test.go

```diff
@@ -0,0 +1,398 @@
+			assert.Nil(t, err)
+			p, _ := New(&lowRiskOverCommitmentArgs, fh)
+
+			preScorePlugin := p.(framework.PreScorePlugin)
+			status := preScorePlugin.PreScore(context.Background(), state, tt.pod, tt.nodes)
+			assert.True(t, status.IsSuccess())
+
+			scorePlugin := p.(framework.ScorePlugin)
+			var actualList framework.NodeScoreList
+			for _, n := range tt.nodes {
+				nodeName := n.Name
+				score, status := scorePlugin.Score(context.Background(), state, tt.pod, nodeName)
+				assert.True(t, status.IsSuccess())
+				actualList = append(actualList, framework.NodeScore{Name: nodeName, Score: score})
+			}
+			assert.EqualValues(t, tt.expected, actualList)
+		})
+	}
+}
+
+var plugin_A *LowRiskOverCommitment = &LowRiskOverCommitment{
+	handle:    nil,
+	collector: nil,
+	args: &pluginConfig.LowRiskOverCommitmentArgs{
+		SmoothingWindowSize: 5,
+		RiskLimitWeights: map[v1.ResourceName]float64{
+			"cpu":    0.5,
+			"memory": 0.5,
+		},
+	},
+	riskLimitWeightsMap: map[v1.ResourceName]float64{
+		"cpu":    0.5,
+		"memory": 0.5,
+	},
+}
+
+var watcherData_A watcher.Data = watcher.Data{
+	NodeMetricsMap: map[string]watcher.NodeMetrics{
+		"node-A": {
+			Metrics: []watcher.Metric{
+				{
+					Type:     watcher.CPU,
+					Operator: watcher.Average,
+					Value:    80,
+				},
+				{
+					Type:     watcher.CPU,
+					Operator: watcher.Std,
+					Value:    0,
+				},
+				{
+					Type:     watcher.Memory,
+					Operator: watcher.Average,
+					Value:    25,
+				},
+				{
+					Type:     watcher.Memory,
+					Operator: watcher.Std,
+					Value:    0,
+				},
+			},
+		},
+	},
+}
+
+var nrla_A *trimaran.NodeRequestsAndLimits = &trimaran.NodeRequestsAndLimits{
+	NodeRequest: &framework.Resource{
+		MilliCPU: 2000,
+		Memory:   2048,
+	},
+	NodeLimit: &framework.Resource{
+		MilliCPU: 3000,
+		Memory:   2048,
+	},
+	NodeRequestMinusPod: &framework.Resource{
+		MilliCPU: 1000,
+		Memory:   1024,
+	},
+	NodeLimitMinusPod: &framework.Resource{
+		MilliCPU: 2000,
+		Memory:   1024,
+	},
+	Nodecapacity: &framework.Resource{
+		MilliCPU: 4000,
+		Memory:   4096,
+	},
+}
+
+var nodeResources_A map[v1.ResourceName]string = map[v1.ResourceName]string{
+	v1.ResourceCPU:    "4000m",
+	v1.ResourceMemory: "4Ki",
+}
+
+var node_A *v1.Node = st.MakeNode().Name("node-A").Capacity(nodeResources_A).Obj()
+
+func TestLowRiskOverCommitment_computeRisk(t *testing.T) {
+	type fields struct {
+		handle              framework.Handle
+		collector           *trimaran.Collector
+		args                *pluginConfig.LowRiskOverCommitmentArgs
+		riskLimitWeightsMap map[v1.ResourceName]float64
+	}
+	type args struct {
+		metrics               []watcher.Metric
+		resourceName          v1.ResourceName
+		resourceType          string
+		node                  *v1.Node
+		nodeRequestsAndLimits *trimaran.NodeRequestsAndLimits
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		args   args
+		want   float64
+	}{
+		{name: "test-cpu",
```

Comment:
nit: put it in a new line is more conventional (applies elsewhere)

```suggestion
		{
			name: "test-cpu",
```

Reply:
Done